linkis-engineconn-plugins/spark/src/main/scala/org/apache/linkis/engineplugin/spark/config/SparkResourceConfiguration.scala

```diff
@@ -31,4 +31,22 @@ object SparkResourceConfiguration {
   val LINKIS_SPARK_EXECUTOR_INSTANCES = CommonVars[Int]("spark.executor.instances", 3)
   val LINKIS_QUEUE_NAME = CommonVars[String]("wds.linkis.rm.yarnqueue", "default")
 
+  val LINKIS_SPARK_KUBERNETES_NAMESPACE =
+    CommonVars[String]("spark.kubernetes.namespace", "default")
```

Comment:
need to be consistent
![image](https://github.com/apache/linkis/assets/11496700/3854c8a6-4915-4518-8f5b-cd577d2d4bf4)